src/ert/scheduler/driver.py

```diff
@@ -237,7 +237,7 @@ async def poll_statuses(self) -> None:
             return
         except ValueError as e:
             # raise this value error as runtime error
-            raise RuntimeError from e
+            raise RuntimeError(e) from e
```

Comment:
I would raise RuntimeError with more metadata to it; eg.: `raise RuntimeError("Why to ValueError?") from e`

Reply:
I wanted to only use `raise RuntimeError(e)` to wrap the message as a RuntimeError, but mypy refused to let it pass linting. Using `raise RuntimeError from e` resulted in `str(exception)` later on being an empty string.